README.md

```diff
@@ -34,14 +36,29 @@ There is a file containing the example calls. Run:
 $ python ./src/code_analyzer/example.py <path-to-your-repo>
 ```
 
+If `test_creation` assists your research, please cite it as follows:
+```bibtex
+@inproceedings{DBLP:conf/recsys/Kula15,
+  author    = {John Shiu and Orix Au Yeung and Tony Shum and Yingzi Jin},
+  title     = {Checklists and LLM prompts for efficient and effective test creation in data analysis},
+  year      = {2024},
+  url       = {http://example.com/paper.pdf},
+}
+
+```
 
 ## Contributing
 
 Interested in contributing? Check out the contributing guidelines. Please note that this project is released with a Code of Conduct. By contributing to this project, you agree to abide by its terms.
 
 ## License
+test_creation is licensed under the MIT License.
 
-`test_creation` was created by John Shiu, Orix Au Yeung, Tony Shum, Yingzi Jin. It is licensed under the terms of the MIT license.
+## Contributors
+John Shiu
+Orix Au Yeung
+Tony Shum
+Yingzi Jin
```

Comment:
This part is redundant as contributors would be shown at the right side of the repository main page. I would suggest remove this.

Reply:
Removed